src/igvfd/schemas/changelogs/donor.md

```diff
@@ -4,6 +4,7 @@
 
 * Add `description`.
 * Add `revoke_detail`.
+* Add `phenotypic_feature`.
```

Comment:
should be plural after last week discussion
phenotypic_features

Reply:
Done